spec/models/manageiq/providers/openstack/infra_manager/refresher_rhos_juno_spec.rb

```diff
@@ -80,7 +80,7 @@ def assert_table_counts
     expect(VmOrTemplate.count).to                be > 0
     expect(OperatingSystem.count).to             be > 0
     expect(Hardware.count).to                    be > 0
-    expect(Disk.count).to                        be > 0
+    expect(Disk.count).to                        eq 0
```

Comment:
Using the OpenStack CLI, I don't see those attributes available from an introspected Ironic node.  I'm not sure this is caused by us.